spec/services/offender_service_spec.rb

```diff
@@ -19,6 +19,6 @@
 
   it "gets a single offender", vcr: { cassette_name: :get_single_offender } do
     offender = OffenderService.new.get_offender('G4273GI')
```

Comment:
Same here - might be worth being explicit that this is a `noms_id` and store in a var